platform/platform.hpp

```diff
@@ -293,6 +294,11 @@ class Platform
   static EConnectionType ConnectionStatus();
   static bool IsConnected() { return ConnectionStatus() != EConnectionType::CONNECTION_NONE; }
 
+  /// Uploads file to the |url| over https.
```

Comment:
`/// Uploads |filePath| to the |url| over https.`?

Reply:
Done.